spec/models/item_spec.rb

```diff
@@ -223,4 +223,27 @@
       expect(create(:item, package_size: '').package_size).to be_nil
     end
   end
+
+  describe "after update" do
+    let(:item) { create(:item, name: "my item", kit: kit) }
+
+    context "when item has kit" do
+      let(:kit) { create(:kit, name: "my kit") }
+
+      it "updates kit name" do
+        item.update(name: "my new name")
+        expect(item.name).to eq kit.name
+      end
+    end
+
+    context "when item does not have kit" do
+      let(:kit) { nil }
+
+      it "does not raise NoMethodError" do
+        expect {
+          item.update(name: "my new name")
+        }.not_to raise_error(NoMethodError)
+      end
+    end
```

Comment:
I don't think this is necessary to check. However, I think a more suitable test to know if this method gets called out at all is stubbing. Make that no instance of `kit` calls the `.update` method.

What do you think?